provider/cmd/pulumi-resource-azure/bridge-metadata.json

```diff
@@ -15035,6 +15048,25 @@
                     }
                 }
             },
+            "azurerm_network_function_azure_traffic_collector": {
+                "current": "azure:network/functionAzureTrafficController:FunctionAzureTrafficController",
+                "past": [
+                    {
+                        "name": "azure:network/functionAzureTrafficController:FunctionAzureTrafficController",
+                        "inCodegen": true,
+                        "majorVersion": 5
+                    }
+                ],
+                "majorVersion": 5,
+                "fields": {
+                    "collector_policy_ids": {
+                        "maxItemsOne": false
+                    },
+                    "virtual_hub_id": {
+                        "maxItemsOne": false
+                    }
+                }
+            },
```

Comment:
To change the token from `azure:network/functionAzureTrafficController:FunctionAzureTrafficController` to `azure:networkFunction/azureTrafficController:AzureTrafficController` without introducing a deprecated resource, you will need to remove this block and regenerate (`make tfgen build_sdks`).